crit/Makefile

```diff
@@ -22,9 +22,13 @@ update-proto:
 	# from other projects, the files of this repository are
 	# now registered under the package name `criu`.
 	ls ./images/*.proto | xargs sed -i '/syntax/a package criu;'
+	mv ./images/sa.proto ./images/criu-sa.proto
+	sed -i 's/sa\.proto/criu-sa\.proto/g' images/*.proto
+	mv ./images/core.proto ./images/criu-core.proto
+	sed -i 's/core\.proto/criu-core\.proto/g' images/*.proto
 
 gen-proto:
-	rm ./images/*.pb.go || true
+	rm -f ./images/*.pb.go || true
```

Comment:
Will we need both `-f` and `|| true`?

Reply:
Thank you for pointing this out! I've dropped this change.